src/Hipace.cpp

```diff
@@ -638,19 +644,35 @@ Hipace::ExplicitSolveOneSubSlice (const int lev, const int ibox, const amrex::Bo
     m_grid_current.DepositCurrentSlice(m_fields, geom[lev], lev, islice);
     // No FillBoundary because grid current only deposits in the middle of the field
 
-    // Modifies Bx, By, Sx and Sy in the current slice
-    ExplicitSolveBxBy(lev);
+    // Modifies Sx and Sy with beam contribution
```

Comment:
```suggestion
    // Set Sx and Sy to beam contribution
```

I understand that's what is actually does, modifies is a bit too general and could be ambiguous